migrations/20171207155647_add_booking_id_column.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = knex => Promise.all([
```

Comment:
Have you thought about how existing records will behave once this column is added. Do you need a script that writes -1 to all existing records as part of this migration?

Reply:
Ahhh you're right!... In normal circumstance I would say yes.. but I belive that Prod will have it's data backed up and the database will be wiped clean moving forward